test/test_fx_passes.py

```diff
@@ -214,7 +214,7 @@ class TestFXGraphPasses(JitTestCase):
         (TestPartitionFunctions.forward12, [["add_2"], ["add_3", "add_4", "add_1"], ["add"]]),
 
         # 5 getitem special case
-        (TestPartitionFunctions.forward13, [["add_2", "add_1", "add"]]),
+        (TestPartitionFunctions.forward13, [["add_2", "add_1", "add", "getitem", "getitem_1", "getitem_2", "getitem_3"]]),
```

Comment:
Hi @IvanYashchuk, I think this is an unexpected change. 
getitem node should always be partitioned together with its producer node. 
Across the stack, we have an implicit assumption that module's input and output must be tensor type. This is why we have the special handling logic in the first place.

Reply:
This doesn't look right even for this PR. `getitem` is produced by `split`, which isn't supported by the fusion node?